library/check_kubeconfig.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python
+
+from ansible.module_utils.basic import AnsibleModule
+import os
+
+
+def main():
+    module = AnsibleModule(argument_spec={})
+    kubeconfig_path = os.environ.get("KUBECONFIG")
+    if not kubeconfig_path:
+        module.fail_json(
+            msg="The KUBECONFIG environment variable is not set."
+        )
+
+    if not os.path.isfile(kubeconfig_path):
```

Comment:
usually we really want to check if we can read the file.  Just because a file exists doesn't mean we can read it.

`with open(kubeconfig_path):`

Reply:
OK will take a look.

Reply:
done.